documentation/1. Introduction/1_Introduction.md

```diff
@@ -1,121 +1,68 @@
-IDSA defines the technical foundation and a set of agreements for secure
-and trusted data spaces, where companies of all sizes and industries can
-manage their data assets. The association already counts over one
-hundred and thirty member organizations from twenty countries. The
-interplay of all these organizations as data space participants and
-service providers will deliver on the shared value proposition of
-generating business value from data.
-
-### The purpose and scope of the Rulebook
+The International Data Spaces Association (IDSA) aims to cultivate a vibrant practitioner community and to provide concrete guidance that enables the realisation of data spaces across a range of capabilities and organisational models.
 
-The IDSA Rulebook serves several purposes regarding the development and
-operation of data spaces. The aim is to describe clearly which rules are
-mandatory and which are optional guidelines. This governance framework
-includes functional, technical, operational, and legal dimensions:
+To that end, IDSA develops the Data Space Requirements (the Rulebook), the Reference Architecture Models (RAMs), and complementary implementation and operations guidance. IDSA also engages with international standardisation bodies and open-source initiatives to harmonise and share the knowledge contributed by its members, thereby supporting the global adoption and interoperability of data space technologies and business models.
+ 
+The central objectives for data spaces is the protection of the autonomy of participants, and to enable their agency over data sharing and use. This leads to the establishment of trustworthiness in data sharing. The IDSA Manifesto articulates the fundamental principles that underpin these objectives:
 
-- Guidelines for the functionality of common services are presented as
-    well as the definition, processes, and services of specific roles.
+<span style="color: red;">insert link and/or screenshot to the Manifesto here</span>
 
-- Guidelines how to implement or use a technical artefact of the IDSA.
+These principles provide the foundation for trusted data sharing and for the consequent development of data-driven services and business models.
 
-- Guidelines for the work and collaboration within data services.
+IDSA specifies foundational requirements and implementable reference architectures that enable organisations of all sizes and sectors to offer, discover, negotiate, and consume data-sharing arrangements for their digital assets.
 
-- Guidelines for the legal basis in compliance with the regulatory
-    environment to ensure trust and security.
+### The purpose and scope of the Rulebook
 
-This framework applies to all IDS-related roles and their interaction in
-the specific environment:
+The IDSA Rulebook supports the creation, operation, and growth of data spaces by distinguishing mandatory requirements from optional, value-adding practices. Its scope spans technical, commercial, and legal dimensions:
```

Comment:
Before distinguishing between them, we first need to mention where the mandatory and optional requirements are coming from. I couldn't find such a statement and we should add it somewhere above or  directly here in this paragraph.